cluster/v1beta1/types_placement.go

```diff
@@ -126,8 +126,15 @@ type GroupStrategy struct {
 	// The percentage will divide the placementDecisions to decisionGroups each group has max number of clusters based on the total num of selected clusters and percentage.
 	// ex; for a total 100 clusters selected, ClustersPerDecisionGroup equal to 20% will divide the placement decision to 5 groups each group should have 20 clusters.
 	// Default is having all clusters in a single group.
-	// If the DecisionGroups field defined, it will be considered first to create the decisionGroups then the ClustersPerDecisionGroup will be used to determine the rest of decisionGroups.
+	//
+	// If the DecisionGroups field defined, it will be considered first to create the decisionGroups. The remaining of clusters will then be divided to decisionGroups based on ClustersPerDecisionGroup.
```

Comment:
I think this doc should be put on groupStrategy field rather than here. for ClustersPerDecisionGroup, we just need to doc that it is the max number of clusters a group should have.